specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "experimentName",
+            "in": "path",
+            "description": "The experiment name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "runId",
+            "in": "path",
+            "description": "The id of the run to cancel.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9][\\w-]{0,255}$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The run was successfully cancelled.",
+            "schema": {
+              "$ref": "#/definitions/RunStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Cancel a run with URI": {
+            "$ref": "./examples/cancelRunWithUri.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RunStatus": {
+      "description": "Contains the details of a run.",
+      "type": "object",
+      "properties": {
+        "runId": {
+          "description": "The identifier for a run.",
+          "type": "string",
+          "example": "myexperiment_155000000001_0"
+        },
+        "target": {
+          "description": "Target refers to compute where the job is scheduled for execution.",
+          "type": "string",
+          "example": "local"
+        }
+      }
+    },
+    "RunDefinition": {
+      "type": "object",
+      "properties": {
+        "configuration": {
+          "$ref": "#/definitions/RunConfiguration",
+          "description": "Fully specified configuration information for the run. Even when that information\r\nis contained in configuration files within the project folder, the client collapses\r\nit all and inlines it into the run definition when submitting a run."
+        },
+        "attribution": {
+          "type": "string"
+        },
+        "telemetryValues": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          }
+        },
+        "snapshotId": {
+          "format": "uuid",
+          "description": "Snapshots are user project folders that have been uploaded to the cloud for subsequent\r\nexecution. This field is required when executing against cloud-based compute targets\r\nunless the run submission was against the API endpoint that takes a zipped project folder\r\ninline with the request.",
+          "type": "string"
+        },
+        "sourceCodeDataReference": {
```

Comment:
sourceCodeDataReference should not be documented; it's an experimental feature only available on the Microsoft tenant.

Reply:
done